.github/workflows/hive.yaml

```diff
@@ -16,13 +16,17 @@ env:
 
 jobs:
   run-hive:
-    name: ${{ matrix.simulation }}
+    name: ${{ matrix.name }}
     runs-on: ubuntu-latest
     strategy:
       matrix:
         include:
           - simulation: rpc-compat
+            name: "Hive: rpc-compat tests"
```

Comment:
This ends up being: `Hive / Hive: rpc-compat tests`, so I would remove the second Hive